src/main/java/com/iexec/core/workflow/ReplicateWorkflow.java

```diff
@@ -1,24 +1,35 @@
 package com.iexec.core.workflow;
 
+import com.iexec.common.notification.TaskNotificationType;
 import com.iexec.common.replicate.ReplicateStatus;
 
 import java.util.Arrays;
-import java.util.List;
+import java.util.HashMap;
 
+import static com.iexec.common.notification.TaskNotificationType.*;
 import static com.iexec.common.replicate.ReplicateStatus.*;
 
 
 public class ReplicateWorkflow extends Workflow<ReplicateStatus> {
 
     private static ReplicateWorkflow instance;
+    private HashMap<ReplicateStatus, TaskNotificationType> actionMap = new HashMap<>();
```

Comment:
Juste use the Map interface for the declaration of the variable.

Reply:
Totally right, thanks